flatpak/rustdesk.json

```diff
@@ -26,32 +43,25 @@
       "name": "rustdesk",
       "buildsystem": "simple",
       "build-commands": [
-        "bsdtar -zxvf rustdesk.deb",
-        "tar -xvf ./data.tar.xz",
-        "cp -r ./usr/*  /app/",
+        "bsdtar -Oxf rustdesk.deb data.tar.xz | bsdtar -xf -",
+        "cp -r usr/* /app/",
         "mkdir -p /app/bin && ln -s /app/lib/rustdesk/rustdesk /app/bin/rustdesk",
-        "mv /app/share/applications/rustdesk.desktop /app/share/applications/com.rustdesk.RustDesk.desktop",
```

Comment:
Without this change, where is the desktop icon?

Reply:
@rustdesk in the `.deb` package. 

ref: 
1) https://github.com/flathub/flathub/pull/5670#discussion_r1788043449
2) https://github.com/flathub/flathub/pull/5670#discussion_r1788042435

Reply:
As far as I know, 24.08 doesn't have any changes for icons. The flatpak builder just takes the icon and `.desktop` from the `.deb` package. It works on 23.08 too.